bitcoin/Cargo.toml

```diff
@@ -42,7 +42,7 @@ hex_lit = "0.1.1"
 
 base64 = { version = "0.13.0", optional = true }
 # Only use this feature for no-std builds, otherwise use bitcoinconsensus-std.
-bitcoinconsensus = { version = "0.20.2-0.5.0", default-features = false, optional = true }
+bitcoinconsensus = { version = "0.20.2-0.5.0", default-features = false, optional = true } # If you change this fix doc links in validation.rs
```

Comment:
this comment is confusing imo.  Also couldn't it go above instead of after the line in question?

Reply:
I was trying to separate the code comments that are meant for consumers of the lib with code comment meant for developers of the lib.

How could I say it more clearly or said another way, what is confusing about "fix the doc links in validation.rs"?

Reply:
I just didn't know what `validation.rs` was.  Maybe the comment could explain validation.rs is a file in `consensus` and that the links contain a version number that will need to match version changes in this line?  Not knowing anything about the consensus lib the comment was confusing at first glance is all.

Reply:
It looks like you removed the comment. Could you please put it back?

Reply:
Oh, woops. Its back in as it was. Thanks

Reply:
> FWIW I thought the comment was perfectly clear. Because it referenced a .rs file I think any external reader would know to just ignore it.

Yeah I guess you can follow the bread crumbs to `validation.rs` and piece together what is meant :detective:

Reply:
Just my own brain damage that thinks it's confusing.  Now that I look again it's pretty clear actually.